pair_practice.rb

```diff
@@ -1,3 +1,5 @@
 class Sentient 
-  
+  5.times do
```

Comment:
do we need initialize?

Reply:
I guess we do need Initialize.

Let's scrap this.